inc/macro_utils/macro_utils.h

```diff
@@ -209,6 +209,12 @@ const char* MU_C3(MU_, enumIdentifier,_ToString)(enumIdentifier value)
 
 #define MU_ENUM_TO_STRING_2(enumIdentifier, value) MU_C3(MU_, enumIdentifier,_ToString)(value)
 
+/*PRI_BOOL is the format specifier that prints a bool/_Bool*/
+#define PRI_BOOL "s (%d)"
+
+/*MU_BOOL_VALUE is the counterpart of PRI_BOOL*/
+#define MU_BOOL_VALUE(v) ((const char*[]){"false", "true"})[!!v], (v)
```

Comment:
>((const char*[]){"false", "true"})[!!v] [](start = 25, length = 39)

nit: any reason not to just use `(v ? "true" : "false")` ? Just seems a tiny bit more readable IMO

Reply:
(I like that you are adding this though :) , I feel like I'm always doing it manually/inconsistently in code)

---
In reply to: [602489608](https://github.com/Azure/macro-utils-c/pull/68#discussion_r602489608) [](ancestors = 602489608)

Reply:
There's 2 reasons:
1. code does not branch. No branches > ternary operator.
2. coverage is unaffected by this. To reach proper coverage with ?: operator at least 2 values would have to be provided "true" and "false" and see that every branch is taken and if the code survives or crashes. With the proposed construct coverage is unaffected, therefore we don't have to write a test for the "false" and another test for "true" to get coverage.

So that's why it looks so funny.

---
In reply to: [602490026](https://github.com/Azure/macro-utils-c/pull/68#discussion_r602490026) [](ancestors = 602490026,602489608)